go/vt/vttest/vtprocess.go

```diff
@@ -199,16 +205,22 @@ func VtcomboProcess(environment Environment, args *Config, mysql MySQLManager) (
 	if args.VtComboBindAddress != "" {
 		vtcomboBindAddress = args.VtComboBindAddress
 	}
+	grpcBindAddress := "127.0.0.1"
```

Comment:
@gmpify , @frouioui , and @GuptaManan100
Was this intentional? It doesn't seem related to the feature added, is it? And I think it might be causing this issue: https://github.com/vitessio/vitess/issues/17396

Reply:
Yes, I think this is incorrect that we missed. We shouldn't add `127.0.0.1` as the default, since an empty default means it listens on all the addresses, which is I think what we want.